.github/workflows/triage.yml

```diff
@@ -7,8 +7,19 @@ on:
 jobs:
   welcome:
     runs-on: ubuntu-18.04
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: JIT
+            column: Need triage
```

Comment:
```suggestion
            column: Need Triage
```

Reply:
@walterddr no, the entire point of having `column` in the matrix is that the column names are different; compare:
- https://github.com/pytorch/pytorch/projects/13
- https://github.com/pytorch/pytorch/projects/17

Reply:
Ahh. LOL.. dunno why we have 2 different naming convention then. but fine.

Reply:
@walterddr yeah I wish they were the same 😅 as you can see in the first commit of this PR, I initially thought they were, but then I had to add a second commit when I realized that they are not